applications/luci-app-statistics/po/zh-cn/statistics.po

```diff
@@ -336,15 +336,15 @@ msgstr "忽略源地址"
 
 #: applications/luci-app-statistics/luasrc/model/cbi/luci_statistics/iptables.lua:103
 msgid "Incoming interface"
-msgstr "入接口"
+msgstr "传入接口"
```

Comment:
The original translation is okay, maybe even more concise.  The same goes for "Outgoing interface"